internal/admin/dashboard/static/js/dashboard.js

```
@@ -534,13 +573,250 @@ function dashboard() {
+        },
+
+        usageLogPrevPage() {
+            if (this.usageLog.offset > 0) {
+                this.usageLog.offset = Math.max(0, this.usageLog.offset - this.usageLog.limit);
+                this.fetchUsageLog(false);
+            }
+        },
+
+        usageLogModelOptions() {
+            const set = new Set();
+            this.modelUsage.forEach(m => set.add(m.model));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)